### PR TITLE
Add flip 1.3

### PR DESCRIPTION
**Pull request by Vertexwahn**
Add flip 1.3

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (flip) have been updated in this PR. Please review the changes.

**Comment by Vertexwahn** [comment]
Need `skip-url-stability-check` label.

File: modules/flip/1.3/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+@@ -0,0 +1,25 @@
++"""
++    SPDX-FileCopyrightText: Copyright 2024 Julian Amann <dev@vertexwahn.de>
++    SPDX-License-Identifier: BSD-3-Clause
++"""
++
++cc_library(
++    name = "flip",
++    hdrs = ["cpp/FLIP.h"],
++    includes = ["cpp"],
```
Comment (fmeum) [inline modules/flip/1.3/patches/add_build_file.patch:12]: These targets aren't public.

**Review by fmeum — CHANGES_REQUESTED** [review]

**Comment by bazel-io** [comment]
Hello @dzbarsky, modules you maintain (aws-in-a-box) have been updated in this PR. Please review the changes.

**Comment by Vertexwahn** [comment]
@dzbarsky Changing aws-in-a-box was by accident - force pushed to get rid of this

**Comment by Vertexwahn** [comment]
@fmeum Ready to go!

**Review by fmeum — APPROVED** [review]

**Pull request merged**